tap-gui/troubleshooting.md

```diff
@@ -238,4 +237,30 @@ app_config:
     reading:
       allow:
       - host: acc-server.accelerator-system.svc.cluster.local
-```
\ No newline at end of file
+```
+
+## <a id='source-scan-no-cves'></a> Source Scan stage may not display CVEs
+
+### Symptom
+
+Source Scan stage vulnerabilities table displays "No records to display" even though there are CVEs in the source scan.
+
+![Screenshot of source scan stage not showing any CVEs](images/troubleshooting-no-cves-source-scan.png)
+
+### Solution
+
+If no CVEs are displayed, run `kubectl describe sourcescan <workload name> -n <workload namespace>` to see if any CVEs were found during the source scan. If so, use the tanzu insight CLI plug-in to query for vulnerability details.  See [Query vulnerabilities, images, and packages](../cli-plugins/insight/query-data.md).
+
+
```

Comment:
I see what you mean @RichardJJG. It looks like @andreizimin's branch may have been based on a version that still had the source scan instructions. This section could be deleted